examples/06_manipulating_images/plot_mask_computation.py

```diff
@@ -19,9 +19,12 @@
 
 """
 
-from nilearn.maskers import NiftiMasker
 import nilearn.image as image
-from nilearn.plotting import plot_roi, plot_epi, show
+
+# Load Miyawaki dataset
```

Comment:
The comment wasn't useful before, but now it is confusing since it was moved away from the actual usage of the fetcher. I'd remove it...

Reply:
move all the imports and removed the comment